examples/tests/valid/redis.w

```diff
@@ -18,11 +18,11 @@ queue.setConsumer(inflight (message: str) => {
 }, timeout: 3s);
 
 test "testing Redis" {
-  // Using raw client
-  let connection = r.rawClient();
-  connection.set("wing", "does redis");
-  let value = connection.get("wing");
-  assert(value == "does redis");
+  // // Using raw client
+  // let connection = r.rawClient();
+  // connection.set("wing", "does redis");
+  // let value = connection.get("wing");
+  // assert(value == "does redis");
```

Comment:
Should I just remove this?

Reply:
I'm good with removing the raw client. If people really want unchecked access to redis within wing, using extern would basically give the same experience (if not better)

Reply:
Lets do it